users/postgres/init.go

```diff
@@ -63,9 +63,9 @@ func migrateDB(db *sqlx.DB) error {
 			{
 				Id: "users_3",
 				Up: []string{
-					`CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+					`CREATE EXTENSION IF NOT EXISTS "pgcrypto";
 					ALTER TABLE IF EXISTS users ADD COLUMN IF NOT EXISTS
-					id UUID DEFAULT uuid_generate_v4()`,
+					id UUID DEFAULT gen_random_uuid()`,
```

Comment:
Before I approve, can you just set `id` column to NOT NULL (`id UUID NOT NULL DEFAULT uuid_generate_v4()`), as well? We didn't include this change to any release, so it's safe to change this migration instead of adding another one.

Reply:
Good point!